group/static/group/js/jquery.quicksearch.js

```diff
@@ -0,0 +1,185 @@
+            return this.matchedResultsCount;
+        };
+
+        this.stripe = function() {
+
+            if (typeof options.stripeRows === "object" && options.stripeRows !== null) {
+                var joined = options.stripeRows.join(' ');
+                var stripeRows_length = options.stripeRows.length;
+
+                jq_results.not(':hidden').each(function(i) {
+                    $(this).removeClass(joined).addClass(options.stripeRows[i % stripeRows_length]);
+                });
+            }
+
+            return this;
+        };
+
+        this.strip_html = function(input) {
+            var output = input.replace(new RegExp('<[^<]+\>', 'g'), "");
+            output = $.trim(output.toLowerCase());
+            return output;
+        };
+
+        this.results = function(bool) {
+            if (typeof options.noResults === "string" && options.noResults !== "") {
+                if (bool) {
+                    $(options.noResults).hide();
+                } else {
+                    $(options.noResults).show();
+                }
+            }
+            return this;
+        };
+
+        this.loader = function(bool) {
+            if (typeof options.loader === "string" && options.loader !== "") {
+                (bool) ? $(options.loader).show(): $(options.loader).hide();
+            }
+            return this;
+        };
+
+        this.cache = function() {
+
+            jq_results = $(target);
+
+            if (typeof options.noResults === "string" && options.noResults !== "") {
+                jq_results = jq_results.not(options.noResults);
+            }
+
+            var t = (typeof options.selector === "string") ? jq_results.find(options.selector) : $(target).not(options.noResults);
+            cache = t.map(function() {
+                return e.strip_html(this.innerHTML);
+            });
+
+            rowcache = jq_results.map(function() {
+                return this;
+            });
+
+            /*
+             * Modified fix for sync-ing "val".
+             * Original fix https://github.com/michaellwest/quicksearch/commit/4ace4008d079298a01f97f885ba8fa956a9703d1
+             * */
+            val = val || this.val() || "";
+
+            return this.go();
+        };
+
+        this.trigger = function() {
+            this.loader(true);
+            options.onBefore();
+
+            window.clearTimeout(timeout);
+            timeout = window.setTimeout(function() {
+                e.go();
+            }, options.delay);
+
+            return this;
+        };
+
+        this.cache();
+        this.results(true);
+        this.stripe();
+        this.loader(false);
+
+        return this.each(function() {
+
+            /*
+             * Changed from .bind to .on.
+             * */
+            $(this).on(options.bind, function() {
+
+                val = $(this).val();
+                e.trigger();
+            });
+        });
+
+    };
+
+}(jQuery, this, document));
```

Comment:
miss a blank line
But I think maybe this js file can download from bower but this file is small, so I'm not quite sure.